src/cdk/app.ts

```diff
@@ -4,13 +4,40 @@ import { MiraServiceStack, MiraStack } from './stack'
 import { Stack } from '@aws-cdk/core'
 import * as fs from 'fs'
 import * as path from 'path'
-import { getBaseStackName, getBaseStackNameFromParams } from './constructs/config/utils'
+import {
+  getBaseStackName,
+  getBaseStackNameFromParams
+} from './constructs/config/utils'
 import { MiraConfig } from '../config/mira-config'
 // eslint-disable-next-line
-const minimist = require('minimist')
+const minimist = require("minimist");
 
 const args = minimist(process.argv)
 
+interface Prototypable {
+  prototype: Stack
```

Comment:
Wondering if per Mark's mention of the extending CDK.Stack if perhaps this would impact or overlap w defining a type like this?